src/components/usa-button/button.stories.js

```diff
@@ -0,0 +1,47 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-button.twig";
+import {
+  Data,
+  AccentCoolData,
+  AccentWarmData,
+  BaseData,
+  BigData,
+  OutlineInverseData,
+  SecondaryData,
+  UnstyledData,
+} from "./content";
+
+export default {
+  title: "Components/Button",
+  argTypes: {
+    text: {
+      control: "text",
+    },
+  },
+};
+
+const Template = (args) => Component(args);
+
+export const Default = Template.bind({});
+Default.args = Data;
+
+export const AccentCool = Template.bind({});
+AccentCool.args = AccentCoolData;
+
+export const AccentWarm = Template.bind({});
+AccentWarm.args = AccentWarmData;
+
+export const Base = Template.bind({});
+Base.args = BaseData;
+
+export const Big = Template.bind({});
+Big.args = BigData;
+
```

Comment:
I believe you are missing an export for the Outline story (I see the json file for it but no reference here)